### PR TITLE
provide descriptions of each function argument

### PR DESCRIPTION
**Pull request by sprsquish**

### CODE REVIEW

File: src/query.js [hunk 1]
```diff
@@ -32,8 +32,10 @@ var objectAssign = require('object-assign');
  *
  * If two are provided, constructs a Ref() function that, when evaluated, returns a Ref value.
  *
- * @param {string|module:query~ExprArg} ref
+ * @param {string|module:query~ExprArg} ref|cls
+ *   Alone, the ref in path form. Comebined with `id`, must be a class ref.
```
Comment (laurenrother) [inline src/query.js]: *Combined

File: src/query.js [hunk 2]
```diff
@@ -103,8 +111,11 @@ function Var(varName) {
  * See the [docs](https://fauna.com/documentation/queries#basic_forms).
  *
  * @param {module:query~ExprArg} condition
+ *   An expression that returns a boolean.
  * @param {module:query~ExprArg} then
- * @param {module:query~ExprArg} _else
+ *   The expression to run if condition is true.
+ * @param {module:query~ExprArg} else
+ *   The expression to run fi the condition is false.
```
Comment (laurenrother) [inline src/query.js]: *if

File: src/query.js [hunk 3]
```diff
@@ -116,6 +127,7 @@ function If(condition, then, _else) {
  * See the [docs](https://fauna.com/documentation/queries#basic_form).
  *
  * @param {...module:query~ExprArg} args
+ *   A seriese of expressions to run.
```
Comment (laurenrother) [inline src/query.js]: *series

File: src/query.js [hunk 4]
```diff
@@ -396,7 +437,9 @@ function Paginate(set, opts) {
  * See the [docs](https://fauna.com/documentation/queries#read_functions).
  *
  * @param {module:query~ExprArg} ref
+ *   An expression resulting in a Ref.
  * @param {?module:query~ExprArg} ts
+ *   The snapshot time at which to check for the instance's existance.
```
Comment (laurenrother) [inline src/query.js]: *existence

File: src/query.js [hunk 5]
```diff
@@ -473,8 +527,11 @@ function Insert(ref, ts, action, params) {
  * See the [docs](https://fauna.com/documentation/queries#write_functions).
  *
  * @param {module:query~ExprArg} ref
+ *   The Ref of the instance whos event should be removed.
```
Comment (laurenrother) [inline src/query.js]: *whose

File: src/query.js [hunk 6]
```diff
@@ -508,6 +569,13 @@ function CreateDatabase(params) {
  * See the [docs](https://fauna.com/documentation/queries#write_functions).
  *
  * @param {module:query~ExprArg} params
+ *   An object of parameters used to create an index.
+ *     - name (required): the name of the index to create
+ *     - source:	One or more source objects describing source classes and (optional) field bindings.
```
Comment (laurenrother) [inline src/query.js]: Weird space here. Intentional?

File: src/query.js [hunk 7]
```diff
@@ -543,6 +617,7 @@ function CreateFunction(params) {
  * See the [docs](https://fauna.com/documentation/queries#sets).
  *
  * @param {module:query~ExprArg} ref
+ *   The Ref of the instance for which to retreive the singleton set.
```
Comment (laurenrother) [inline src/query.js]: *retrieve

File: src/query.js [hunk 8]
```diff
@@ -553,7 +628,8 @@ function Singleton(ref) {
 /**
  * See the [docs](https://fauna.com/documentation/queries#sets).
  *
- * @param {module:query~ExprArg} ref_set
+ * @param {module:query~ExprArg} ref
+ *   A Ref or SetRef to retreive an event set from.
```
Comment (laurenrother) [inline src/query.js]: *retrieve

File: src/query.js [hunk 9]
```diff
@@ -637,7 +721,10 @@ function Join(source, target) {
  * See the [docs](https://fauna.com/documentation/queries#auth_functions).
  *
  * @param {module:query~ExprArg} ref
+ *   A Ref with credentials to autenticate against
```
Comment (laurenrother) [inline src/query.js]: *authenticate

File: src/query.js [hunk 10]
```diff
@@ -953,7 +1071,9 @@ function Equals() {
  * See the [docs](https://fauna.com/documentation/queries#misc_functions).
  *
  * @param {module:query~ExprArg} path
- * @param {module:query~ExprArg} _in
+ *   An array representing a path to check for the existance of.
```
Comment (laurenrother) [inline src/query.js]: *existence

**Comment by laurenrother** [comment]
👍

File: src/query.js [hunk 11]
```diff
@@ -156,7 +169,7 @@ var objectFunction = function(fields) {
  * Directly produces a FaunaDB Lambda expression as described in the FaunaDB reference
  * documentation.
  *
- * @param {module:query~ExprArg} var_name
+ * @param {module:query~ExprArg} var
```
Comment (marrony) [inline src/query.js]: I think this should match the function arg name, doesn't the linter complained about it?

**Pull request merged**